tox.ini

```diff
@@ -9,7 +10,18 @@ deps =
 commands =
 	py.test {posargs}
 
+[testenv:negtz]
+passenv = LANG
+setenv =
+    TZ = America/Argentina/Buenos_Aires
+deps =
```

Comment:
Yup, you're right: https://testrun.org/tox/latest/config.html#substitution-for-values-from-other-sections

I'll look into it later today.